src/surforama/gui/qt_point_io.py

```diff
@@ -63,9 +123,26 @@ def _load_star_file(self, file_path: Path):
         )
 
         # get the vectors data
-        normal_data, up_data = vectors_data_from_points_data(
-            point_coordinates=point_coordinates, features_table=features_table
-        )
+        if features_table is None:
```

Comment:
in case no normal and up-vectors can be read from the star file, we use the normal vector from the mesh and initialize a random up-vector.